index.html

```diff
@@ -84,197 +46,274 @@ <h4>Zero-ops OpenStack on just about any Linux box.</h4>
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" value="sudo snap install microstack --classic --edge" readonly="readonly">
+              <button class="p-code-copyable__action">Copy to clipboard</button>
+            </div>
 
-          <h3>Configure networks and OpenStack databases</h3>
+            <p class="p-stepped-list__content u-stepped-list-margin">Don’t have the <code>snap</code> command? <a
+                href="https://docs.snapcraft.io/installing-snapd" class="p-link--external">Get set up for snaps.</a></p>
+          </li>
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Configure networks and OpenStack databases</h3>
 
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" aria-label="sudo microstack.init" value="sudo microstack.init --auto" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" value="sudo microstack.init --auto" readonly="readonly">
+              <button class="p-code-copyable__action">Copy to clipboard</button>
+            </div>
+          </li>
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Launch a VM</h3>
 
-          <p>
-            That’s it! Your OpenStack is ready.
-          </p>
-        </div>
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" value="microstack.launch cirros --name test" readonly="readonly">
+              <button class="p-code-copyable__action">Copy to clipboard</button>
+            </div>
+            <p>Your VM is up and running now. You should see a message like:</p>
+            <p>Access your server with <code class="highlighter-rouge">ssh -i /home/ubuntu/.ssh/id_microstack cirros@10.20.20.3</code></p>
+          </li>
+        </ol>
+        <p>
+          That’s it! Your OpenStack is ready.
+        </p>
       </div>
-      <div class="u-fixed-width">
-        <hr class="p-separator" />
+      <div class="col-6 js-tab__content" id="tab-two__content" role="tabpanel" aria-labelledby="tab-two">
+        <ol class="p-stepped-list u-no-margin--left">
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Install Multipass for Windows</h3>
+
+            <p class="p-stepped-list__content u-stepped-list-margin">
+              <a href="https://github.com/CanonicalLtd/multipass/releases" class="p-button--positive js-download"
+                data-os="win-win">
+                <span class="p-link--external">Download Multipass for Windows</span>
```

Comment:
```suggestion
                Download Multipass for Windows
```
As long as the JavaScript works, this doesn’t send you to a different site, so it shouldn’t be presented as an external link.